hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformations.scala

```diff
@@ -270,7 +270,7 @@ object DemographicsTransformations {
       if (allActivities.contains[Long](ActivityValues(activity))) {
         rawRecord.getOptionalNumber(s"dd_${activity}_m").orElse(Some(3L))
       } else {
-        None
```

Comment:
I think a brief comment explaining this would be good

Reply:
seconding the above, but i also think we can simplify this, since the only difference between these two branches is the value in `orElse`.